DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java

```diff
@@ -38,7 +36,7 @@ public void handle(String target,
     // used to start the CI server in command line
     public static void main(String[] args) throws Exception
     {
-        Server server = new Server(8080);
+        Server server = new Server(80);
```

Comment:
Is there a reason for the change of port?

Reply:
I agree, shouldn't it follow the convention from the Canvas assignment, aka. port 8000+<group number> so 8012 in our case

Reply:
Ah, that’s left over from when I tested the feature, and I forgot to change it back. 
(When I tried to start ngrok from port 8080 it said the port was ”already in use”.) I’ll fix it.